src/AltGoslingComponent.tsx

```diff
@@ -170,21 +171,24 @@ export const AltGoslingComponent = (props: AltGoslingCompProps) => {
             // Get AltGoslingSpec
             const altSpec = getAlt(specProcessed);
             // Set dimensions
-            setGoslingDimensions({width: specProcessed._assignedWidth, height: specProcessed._assignedHeight});
+            setGoslingDimensions({ width: specProcessed._assignedWidth, height: specProcessed._assignedHeight });
             // Update current alt
             updateAltPanelDisplay(altSpec);
             // Reset data panels
             setDataPanelCurrent(undefined);
             setDataPanelPrevious(undefined);
             // setExpandedAltPanelWrapper(['tree']);
             // setExpandedDataPanelWrapper(['tree']);
+
+            // Callback function to pass over the AltGoslingSpec
+            props.onAltGoslingSpecUpdate?.(altSpec);
```

Comment:
A place to call the onAltGoslingSpecUpdate

Reply:
Do we want to add this to the [AltGoslingCompProps](https://github.com/gosling-lang/altgosling/blob/f6b993bf6ccd0a70d6462a3ef7b19630379a7aa6/src/AltGoslingComponent.tsx#L35)?

Reply:
Yes, I already added it as well